spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -8,37 +8,189 @@ def it_redirects_to_user_login
     it_redirects_to_simple new_user_session_path
   end
 
-  describe 'new' do
-    context 'without javascript' do
-      it 'should not return the form for anyone not logged in' do
+  describe "new" do
+    context "without javascript" do
+      it "redirects anyone not logged in" do
         get :new
         it_redirects_to_user_login
       end
 
-      it 'should render the form if logged in' do
+      it "renders the new form template if logged in" do
         fake_login
         get :new
-        expect(response).to render_template('new')
+        expect(response).to render_template("new")
       end
     end
 
-    context 'with javascript' do
-      it 'should render the bookmark_form_dynamic form if logged in' do
+    context "with javascript" do
+      it "renders the bookmark_form_dynamic form if logged in" do
         fake_login
         get :new, params: { format: :js }, xhr: true
-        expect(response).to render_template('bookmark_form_dynamic')
+        expect(response).to render_template("bookmark_form_dynamic")
       end
     end
   end
 
-  describe 'edit' do
-    context 'with javascript' do
+  describe "edit" do
+    context "with javascript" do
       let(:bookmark) { FactoryBot.create(:bookmark) }
 
-      it 'should render the bookmark_form_dynamic form' do
+      it "renders the bookmark_form_dynamic form if logged in" do
         fake_login_known_user(bookmark.pseud.user)
         get :edit, params: { id: bookmark.id, format: :js }, xhr: true
-        expect(response).to render_template('bookmark_form_dynamic')
+        expect(response).to render_template("bookmark_form_dynamic")
+      end
+    end
+  end
+
+  describe "index" do
+    let!(:external_work_bookmark) { create(:external_work_bookmark) }
+    let!(:series_bookmark) { create(:series_bookmark) }
+    let!(:work_bookmark) { create(:bookmark) }
+
+    it "returns search results when given bookmark_search parameters" do
```

Comment:
Rereading this again, I don't think this is testing ES. The bookmark shows up because it's in `Bookmark.latest`.

Reply:
Yeah, you're right -- I switched it to checking for the series bookmark (which wouldn't by in Bookmark.latest) and that didn't turn up.